app/templates/components/-report/metrics-chart/summary.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="row">
+  <div class="col">
+    Metric totals
+  </div>
+  <div class="col">
+    <div class="row">
+      {{#each metrics as |metric|}}
+        <div class="col text-nowrap" title="{{ metric.total }}">
+          {{metric.label}}: {{number-format metric.total "0.[00]a"}}
```

Comment:
I would reduce the padding and remove the columns. Just display the numbers in line and aligned left. No need to have them in columns or split to the right.

Reply:
![image](https://user-images.githubusercontent.com/1778222/112691916-09529d80-8e4c-11eb-8ec4-52b636bd6287.png)